### DISCUSSION TITLE
Point support for mongodb

### DISCUSSION THREAD

**Discussion by Hellomik2002** [prisma/prisma#21046] [Q&A]
Mongo db has special type for Points, 
```js
{
type: "Point",
cordinates: []
}
```
Can you add it? to work with coordinates or add custom types without creating collection for model

**Comment by ludralph** [comment]
Hi @Hellomik2002 👋 

Thank you for raising this question.

We currently do not have Geospatial support but there is an open feature request for supporting it.

- https://github.com/prisma/prisma/issues/1798

It would be helpful if you can add your usecase to the issue and leave a 👍 to help our engineers prioritize it.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
